content/en/docs/instrumentation/erlang/testing.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Testing"
+draft: true
```

Comment:
remove the draft and put a page weight here so that you have a defined position in the navigation